_src/op.adoc

```diff
@@ -0,0 +1,15 @@
+include::index.adoc[]
+////
+NOTE: this file exists to allow URLs to https://sepheo.co/op to reach
+the landing page content.
```

Comment:
I don't follow. What's the purpose of `https://sepheo.co/op`?

Reply:
When we bought the domain, we looked at a .coop domain and it seemed too expensive.  But then we thought, if you put a page called 'op' on a `.co` domain, you have `.co/op`.  A pauper's .coop domain!

This particular file is just trying to preserve this feature, of an `op` page which is identical to the landing page, that we had in the earlier version.

Reply:
Although the link seems to have stopped working in the old page...